assets/src/edit-story/components/canvas/page.js

```diff
@@ -39,9 +40,16 @@ const Element = styled.div`
 `;
 
 function Page() {
+	const selectionRef = createRef();
+	const [ targetEl, setTargetEl ] = useState( null );
+
+	useEffect( () => {
+		setTargetEl( selectionRef.current );
```

Comment:
I noticed this in couple of places: is there something bad from just doing call ref-callback? E.g.

```
<X ref={current => setTargetEl(current)}
```

Reply:
Hmm, I'm not sure if there's a difference/downside doing that. ^  @barklund Any thoughts here?

Reply:
Usually the main downside to using a ref-callback is that you don't have `ref.current` handy. But since we're writing the value into an easily-available state, we will have it anyway. So I think there's no real downside that I can see at this point. You'd remove an extra re-render this way vs `useEffect -> setState` approach. Not a huge deal, but still. Also the code itself would actually be simpler: `<X ref={setTargetEl}>` - so it looks pretty natural, imho.

Reply:
👍 Will update this and then merge this PR.